lib/resolver.content.js

```diff
@@ -227,6 +252,7 @@ export default class PodletClientContentResolver {
             if (outgoing.redirectable && statusCode >= 300) {
                 outgoing.redirect = {
                     statusCode,
+                    // @ts-expect-error TODO: look into what happens if the podlet returns more than one location header
```

Comment:
TS highlighted this as a potential error, but I didn't want to make a decision there now